lib/kdump_utils.pm

```diff
@@ -379,6 +379,28 @@ sub check_function {
     }
 }
 
+# for bsc#1199326, we need to check if ~/bernhard/.ssh/id_rsa was not
+# affected after kdump_and_crash.
+sub check_ssh_files {
+    # if file ~bernhard/.ssh/id_rsa missing or zero size
+    # we need to recreate the ssh key.
+    my $ret1 = script_run('! test -e ~bernhard/.ssh/id_rsa');
+    my $ret2 = script_run('! test -s ~bernhard/.ssh/id_rsa');
```

Comment:
I think the '-s' is enough for file existing and with content, no need to use '-e' any more. WDYT?

Reply:
it's include 2 kind of scenarios. 1 file not existed, the other is file existed but with zero size (null)

Reply:
The condition is '!' for 'file existed and with content', so I think the ''! test -s ~bernhard/.ssh/id_rsa' is enough for the two scenarios.